lib/transform/pathRegexTransformer.ts

```diff
@@ -111,6 +111,9 @@ export const pathRegexTransformer: SpecTransformer = {
     traverseSwagger(spec, {
       onPath: (path, pathTemplate) => {
         let pathStr = pathTemplate;
+        if ([...pathStr.matchAll(/\{/g)].length !== [...pathStr.matchAll(/\}/g)].length) {
```

Comment:
It would be better to only report this error in semantic validation. Can we move this code to there?